rosdep/python.yaml

```diff
@@ -1657,6 +1657,11 @@ python-netaddr:
     trusty_python3: [python3-netaddr]
     utopic: [python-netaddr]
     vivid: [python-netaddr]
+    xenial: [python-netaddr]
+    yakkety: [python-netaddr]
+    zesty: [python-netaddr]
+    artful: [python-netaddr]
```

Comment:
sorry I should have specified: the entries have to be in alphabetical order

Reply:
ha I forgot as well, sorry I ll fix that.